src/main/java/org/openrewrite/java/migrate/apache/commons/lang/StringUtilsMethodToJdkTemplate.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.apache.commons.lang;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.openrewrite.*;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.JavaVisitor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.tree.J;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class StringUtilsMethodToJdkTemplate extends Recipe {
+
+    @Option(displayName = "Method",
+            description = "The Apache Commons Lang `StringUtils` method to convert to a JDK equivalent.",
+            example = "org.apache.commons.lang3.StringUtils isBlank(java.lang.String)")
+    String methodPattern;
+
+    @Option(displayName = "JDK template",
+            description = "The JDK method to convert to.",
+            example = "#{any(String)}.isBlank()")
+    String jdkTemplate;
+
+    @Option(displayName = "Imports",
+            description = "The imports to add to the compilation unit.",
+            example = "java.util.Objects",
+            required = false)
+    String[] imports;
```

Comment:
I think that ideally `jdkTemplate` should accept the fully qualified name so that imports are not needed explicitly. Let me know if I am missing anything